Automattic/jetpack
# Search: fix widget setting not working for WordPress 5.8

Fixes #20055

#### Changes proposed in this Pull Request:
* Changed the selector to determine whether the widget is Jetpack Search widget or not so that the widget setting works in WP 5.8.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Ensure widget setting still works under WP 5.7 (Jetpack Dev Env)
* Go to a WP 5.8 site with Jetpack Search subscription (I just upgraded my local WP using the beta tester plugin [here](https://wordpress.org/news/2021/06/wordpress-5-8-beta-1/))
* Open Customizer / Widget Editor
* Ensure you can add Jetpack Search widget and add/remove filters and successfully save settings.

Note: After testing this, I run `jetpack docker uninstall` and `jetpack docker install` to get the local env back - but all previous data would be lost, please do an export before upgrade to WP 5.8 if you'd need the local data.
